zstacklib/zstacklib/utils/pci.py

```
@@ -235,10 +235,9 @@ def get_pci_device_names():
 
 
 def collect_pci_devices_with_dependencies(pciDeviceAddress):
-    pcis = []
-    device_path = os.path.join("/sys/bus/pci/devices/", pciDeviceAddress)
-    for item in os.listdir(device_path):
-        if item.startswith("consumer:pci:"):
-            dependent_addr = item.replace("consumer:pci:", "")
-            pcis.append(dependent_addr)
-    return pcis
+    base_address = pciDeviceAddress.rsplit('.', 1)[0]
+    r, o, e = bash_roe("ls /sys/bus/pci/devices/ | grep %s" % base_address)
```

Ruff (0.14.10)

Unpacked variable `e` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)